bin/update

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env ruby
+require 'fileutils'
+
+# path to your application root.
+APP_ROOT = File.expand_path('..', __dir__)
+
+def system!(*args)
+  system(*args) || abort("\n== Command #{args} failed ==")
+end
+
+raise "Only use bin/update in Development environment" if ENV["RAILS_ENV"] == "test" || ENV["RAILS_ENV"] == "production"
```

Comment:
I think this could probably be more safely generalized in case we add a demo / staging environment:

```suggestion
raise "Only use bin/update in Development environment" if ENV["RAILS_ENV"] && ENV["RAILS_ENV"] != "development"
```